internal/svc/admin_svc.go

```diff
@@ -102,6 +104,81 @@ func (cas *CerbosAdminService) AddOrUpdateSchema(ctx context.Context, req *reque
 	return &responsev1.AddOrUpdateSchemaResponse{}, nil
 }
 
+func (cas *CerbosAdminService) ListSchemas(ctx context.Context, req *requestv1.ListSchemasRequest) (*responsev1.ListSchemasResponse, error) {
+	if err := cas.checkCredentials(ctx); err != nil {
+		return nil, err
+	}
+
+	if cas.store == nil {
+		return nil, status.Error(codes.NotFound, "store is not configured")
+	}
+
+	schemaIds, err := cas.store.ListSchemaIDs(ctx)
+	if err != nil {
+		return nil, status.Error(codes.NotFound, fmt.Sprintf("failed to list schema ids: %s", err.Error()))
+	}
+
+	sortSchemas(schemaIds)
+
+	return &responsev1.ListSchemasResponse{
+		SchemaIds: schemaIds,
+	}, nil
+}
+
+func (cas *CerbosAdminService) GetSchema(ctx context.Context, req *requestv1.GetSchemaRequest) (*responsev1.GetSchemaResponse, error) {
+	if err := cas.checkCredentials(ctx); err != nil {
+		return nil, err
+	}
+
+	if cas.store == nil {
+		return nil, status.Error(codes.NotFound, "store is not configured")
+	}
+
+	schemas := make([]*schemav1.Schema, 0, len(req.Id))
+	for _, id := range req.Id {
+		sch, err := cas.store.LoadSchema(context.Background(), id)
+		if err != nil {
+			return nil, status.Error(codes.Internal, fmt.Sprintf("could not get schema with id %s: %s", id, err.Error()))
+		}
+
+		schBytes, err := ioutil.ReadAll(sch)
+		if err != nil {
+			return nil, status.Error(codes.Internal, fmt.Sprintf("could not read schema from interface: %s", err.Error()))
+		}
+
+		schemas = append(schemas, &schemav1.Schema{
+			Id:         id,
+			Definition: schBytes,
+		})
+	}
+
+	return &responsev1.GetSchemaResponse{
+		Schemas: schemas,
+	}, nil
+}
+
+func (cas *CerbosAdminService) DeleteSchema(ctx context.Context, req *requestv1.DeleteSchemaRequest) (*responsev1.DeleteSchemaResponse, error) {
+	if err := cas.checkCredentials(ctx); err != nil {
+		return nil, err
+	}
+
+	ms, ok := cas.store.(storage.MutableStore)
+	if !ok {
+		return nil, status.Error(codes.Unimplemented, "Configured store is not mutable")
+	}
+
+	log := ctxzap.Extract(ctx)
+
+	for _, id := range req.Id {
```

Comment:
Sorry, I didn't notice this before: wouldn't this result in N database queries? We can change it to a single query by making the `DeleteSchema` method accept a variable list of IDs.